content/sensu-go/6.6/operations/deploy-sensu/generate-certificates.md

```diff
@@ -233,17 +232,14 @@ You should now have a set of files for use by Sensu agents:
 filename           | description                  | required on agent?  |
 -------------------|------------------------------|---------------------|
 `agent.pem`        | Agent certificate            | {{< check >}}       |
-`agent-key.pem`    | Agent private key            | {{< check >}}       |
```

Comment:
Same issue here with the `agent-key.pem` as for the backend. Is this the same as (or copied from) `ca-key.pem`? If so, do we have this incorrectly marked as required?

Reply:
@hillaryfraley `agent-key.pem` is required by the agent's `key-file` param